source/NVDAObjects/JAB/__init__.py

```diff
@@ -229,7 +229,12 @@ def _get_TextInfo(self):
 		return super(JAB,self).TextInfo
 
 	def _isEqual(self,other):
-		return super(JAB,self)._isEqual(other) and self.jabContext==other.jabContext
+		if not isinstance(other,JAB):
+			return False

```

Comment:
nit: This check is redundant, as `NVDAObject.__eq__` (which calls _isEqual) already checks that the types are the same.